server/src/processor/task_allocator.rs

```
@@ -50,6 +50,42 @@ impl<'a> TaskAllocationProcessor<'a> {
                             error = %state_store_error,
                             "Unable to allocate task"
                         );
+
```

Can we fail the task and invocation in create_allocation() call? Do we really need to propagate the error details here?

---

Sure, no problem - done.